cluster/vagrant/provision-minion.sh

```diff
@@ -72,16 +72,17 @@ EOF
 # See: https://github.com/mitchellh/vagrant/issues/2430
 hostnamectl set-hostname ${MINION_NAME}
 
-# Workaround to vagrant inability to guess interface naming sequence
-# Tell system to abandon the new naming scheme and use eth* instead
-rm -f /etc/sysconfig/network-scripts/ifcfg-enp0s3
```

Comment:
I just forgot to add it back to the minion provisioning script.

The VMWare box does not have a enp0s3 interface defined. It also does not have an eth* device. It does have a en33, but ti works just fine.

I'll make sure that we are also deleting the network script on the minions.